docs/source/installation/development.rst

```diff
@@ -3,5 +3,123 @@
+You can find the same for Debian/Ubuntu `here <https://www.tecmint.com/install-postgresql-on-debian-and-ubuntu/>`_.
+
+CentOS/Fedora
++++++++++++++
+
+.. code-block:: shell
+
+    yum install -y gcc redis python-devel python-virtualenv libjpeg-turbo-devel libxslt-devel libxml2-devel \
+        libffi-devel pcre-devel libyaml-devel redhat-rpm-config
+    systemctl start redis.service
+
+
+Debian/Ubuntu
++++++++++++++
+
+.. code-block:: shell
+
+    apt install -y --install-recommends python-dev python-virtualenv libxslt1-dev libxml2-dev libffi-dev libpcre3-dev \
+        libyaml-dev build-essential redis-server
+
+Then on Debian::
+
+    apt install -y libjpeg62-turbo-dev
+
+And on Ubuntu::
+
+    apt install -y libjpeg-turbo8-dev zlib1g-dev
+
+
+Creating the directory structure
+--------------------------------
+
+You will need a directory in your file system to store Indico as well as its data files (archives, etc...). Some
+developers keep all their code inside a ``dev`` or ``code`` dir. We will assume ``dev`` here.
+
+.. code-block:: shell
+
+    mkdir -p ~/dev/indico/data
+
+We will need a virtualenv where to run Indico::
+
+    cd ~/dev/indico
+    virtualenv env -p /usr/bin/python2.7
+
+
+Cloning Indico
+--------------
+
+First, let's clone Indico's code base. If you're going to contribute back to the project, it's probably best if
+you clone your own `GitHub fork of the project <https://help.github.com/articles/fork-a-repo/>`_. Otherwise, if you're
+just thinking of playing a bit with the source code, it's enough if you clone the upstream repo:
+
+.. code-block:: shell
+
+    git clone --recursive https://github.com/indico/indico.git src
+
+If you're going to be changing the standard Indico plugins and/or the documentation, you can also clone those::
+
+    git clone https://github.com/indico/indico-plugins.git plugins
+    git clone https://github.com/indico/indico-user-docs.git user-docs
+
+
+Setting up Maildump (advised)
+-----------------------------
+
+.. code-block:: shell
+
+    virtualenv maildump -p /usr/bin/python2.7
+    . ./maildump/bin/activate
+    pip install maildump
+    maildump -p /tmp/maildump.pid
+
+
+Creating the DB
+---------------
+
+.. code-block:: shell
+
+    sudo -u postgres createuser <your-user>
+    sudo -u postgres createdb -O <your-user> indico
+    sudo -u postgres psql indico -c "CREATE EXTENSION unaccent; CREATE EXTENSION pg_trgm;"
+
+
+Configuring
+-----------
+
+Let's get into the Indico virtualenv::
+
+    . ./env/bin/activate
+
+.. code-block:: shell
+
+    cd src
+    pip install -r requirements.dev.txt
+    pip install -e .
+    fab setup_deps:system_node=1
+
+Then, follow the instructions given by the wizard::
+
+    indico setup wizard --dev
+
+You can then initialize the DB::
+
+    indico db prepare
+
+
+Running Indico
+--------------
+
+.. code-block:: shell
+
+    indico run -h <your-hostname>
```

Comment:
`-q` to not show requests for static files and maybe `--enable-evalex` because it's useful (since there's the PIN it's safe(-ish) even if someone exposes it to more than localhost)